amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -56,6 +56,13 @@ impl Transform {
     // TODO: fix example
     #[inline]
     pub fn look_at(&mut self, target: Vector3<f32>, up: Vector3<f32>) -> &mut Self {
+        /*
+         * Are you sure you actually want the look_at matrix here? It is worth noting this will
+         * generate a look_at view matrix (intended to be used by a camera). So that might
+         * be the inverse of what you intended to do here.
+         * You should try `new_observer_frame` (https://www.nalgebra.org/rustdoc/nalgebra/geometry/type.UnitQuaternion.html#method.new_observer_frame)
+         * instead.
+         */
```

Comment:
I don't actually know what we want here although I assume it's about having the local rotation face `target`.

Reply:
I think the most likely is that you want `new_observer_frame` since the `Transform` is not intended to be used for cameras only. I will make the change.

Reply:
Reading the previous code based on `cgmath`, it turns out the matrix being used is actually a look_at view matrix. I find this weird and makes wonder if this method from `Transform` is actually used anywhere on amethyst. Anyway, to avoid breaking things I will change the code so it matches the old behavior instead of using `new_observer_frame`.